packages/core/src/app/customer/FastllanePrivacySettings.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent, useEffect } from 'react';
+
+import { CheckoutContextProps } from '@bigcommerce/checkout/payment-integration-api';
+import { withCheckout } from '../checkout';
+import { PaymentMethod } from '@bigcommerce/checkout-sdk';
+
+export interface WithCheckoutProps {
+    providerWithCustomCheckout?: string;
+    getPaymentMethod: (methodId: string) => PaymentMethod | undefined;
+}
+
+interface FastlaneWatermarkOptions {
+    includeAdditionalInfo: boolean
+}
+
+interface FastlaneWatermarkComponent {
+    FastlaneWatermarkComponent: (options?: FastlaneWatermarkOptions) => Promise<FastlanePrivacySettings>
+}
+
+declare global {
```

Comment:
I do not think that is good idea to set global variable here.

Would be better to create extended interface from `Window` (interface FastlaneHostWindow extends Window) and add type guard where needed

@serhii-tkachenko what do you think?

Reply:
Fixed